src/amo/components/SearchContextCard/index.js

```diff
@@ -164,6 +164,12 @@ export class SearchContextCardBase extends React.Component<InternalProps> {
     } else if (loadingSearch) {
       searchText = i18n.gettext('Searching for add-ons');
     }
+    if (searchText && tag) {
+      const tagText = i18n.sprintf(i18n.gettext('with tag %(tag)s'), {
+        tag,
+      });
+      searchText = `${searchText} ${tagText}`;
+    }
```

Comment:
We need full translated strings for localizers so we cannot do that unfortunately.

Reply:
And I guess we only want the translation for tags when we do have search results and not when it's loading.

Reply:
😐  there are 20 strings to cover the existing combinations of addonType + query-or-not + category-or-not, for singular and plural.  With tags too it'll be an additional 20.

Having strings for each combination seems suboptimal to me but I could add them.

I considered we could ignore some less likely scenarios - tag pages only show with no category, and no query, so that could be the only scenario with a string mentioning the tag (addonType we'd always need as there's a filter for it) so 6 strings. But adding a `tag=xxx` parameter to a category or search results page then wouldn't mention the tag filter, and it would also apply to a tag page (e.g. `/tag/foo/?q=baa`) that has extra parameters added to the url - if the url is manually edited it's too easy to end up with an unsupported case that we wouldn't know what to do with, especially as `SearchContextCard` is used by search, category, and now tag pages.

Reply:
https://github.com/mozilla/addons-frontend/pull/10820/commits/c848cce9428199024e3755b16eed6342752e90b0 is what it looks like just to cover the cases we directly support with links and/or the UI (ignores combinations with categories or queries that are possible with specific url strings)